osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -141,6 +146,15 @@ private void load(OsuColour colours)
 
             InternalChildren = new Drawable[]
             {
+                lineHighlightBackground = new Container
```

Comment:
Not sure about this existing from the outset. Any reason it can't be added when the highlight actually occurs? Seems like unnecessary overhead.